_data/sections/contribution-limits/legends.yml

```diff
@@ -88,3 +88,8 @@
       max: Infinity
       label: Unlimited
       color: "#cb181d"
+- name: Party2Party
+  type: singular
```

Comment:
This value seems to end up bubbling through and causing this `d3.scaleSingular is not a function` error that I'm seeing.